content/client-server-api/modules/instant_messaging.md

```diff
@@ -135,7 +135,7 @@ between rooms.
 Messages SHOULD appear immediately in the message view when a user
 presses the "send" button. This should occur even if the message is
 still sending. This is referred to as "local echo". Clients SHOULD
-implement "local echo" of messages. Clients MAY display messages in a
+implement "local echo" of messages. Clients might display messages in a
```

Comment:
Why make it in caps even, then? The word in caps calls back to the concern I expressed in #819: as a spec reader, I would try to treat it as a sort of normative, even though the language here doesn't imply any particular "norm" aside from recommending a UI feature. Literally "are encouraged to" is probably best here?

Reply:
I haven't done the archeology here, but it would largely depend on the original MSC/specification work. If the intent wasn't specifically "are encouraged to", then we should have an MSC to change/demote the language whereas with this change we're maintaining implied meaning even if inventing words - it's still normative here.

Reply:
> we should have an MSC to change/demote the language

For me, "are encouraged to" is stronger than "MAY".

Reply:
It seems that we should really spec any best-fitting (up to native speakers) word for the case when we want to explicitly "allow" (in the sense described in #819) certain behaviour, and then use this word. Otherwise we risk reiterating the discussion the next time somebody finds the wording not clear enough. I, for one, am perfectly fine with MAY as well as CAN, although I always perceived CAN as more of technical/physical capability and MAY as "being allowed" (by a normative document, in particular). I'd be somewhat confused by MIGHT as the allowing word but can live with that, too.

Reply:
This thread seems to have got stuck.

@turt2live you might be best placed to unstick it?

Reply:
I'm not sure I can. I've already given my opinion, and am not concerned with the specific outcome here. I would ask that whatever we decide, we document it to prevent it coming up again (per @KitsuneRal's comments).

Reply:
Never has so much discussion taken place over a single word.

The current wording ("MAY") is at least clear, and better than "might", so if you're not concerned, I'll go ahead and close this PR and the original bug.